### ISSUE TITLE
Tooltip of absolute positioned button is placed incorrectly

### ISSUE THREAD

**Issue by anhallbe** [infor-design/enterprise#1300]
**Describe the bug**
When a button (or possibly any element?) is positioned with `position: absolute`, and it has a `soho-tooltip`, the tooltip is sometimes misplaced when the button is clicked and/or moved.

**To Reproduce**
In any component, add the following:
```
// Template:
<button
      *ngFor="let button of buttons"
      soho-button="icon"
      icon="alert"
      soho-tooltip
      [title]="button.text"
      (click)="generateButtons()"
      style="position: absolute; left:50%;"
      style.top="{{button.num * 10}}%"
      ></button>

// Component class:
export class SomeComponent {
  buttons: {text: string, num: number}[];
  constructor() {
    this.generateButtons();
  }

  generateButtons() {
    const numButtons = Math.floor(Math.random() * 10) + 1;
    this.buttons = [];
    for (let i = 0; i < numButtons; i++) {
      this.buttons.push({text: `Button ${i}`, num: i});
    }
  }
}
```
Then:

1. Click a button
2. Keep mouse hovered over that button
3. The tooltip is now in the top left corner of the viewport
4. Move the mouse
5. The tooltip is now correctly placed on the button

**Expected behavior**
The tooltip should always be placed on the button that is hovered.

**Version**
- ids-enterprise-ng: 4.11 & master

**Screenshots**
![absolute_tooltip](https://user-images.githubusercontent.com/4991954/48951263-30c49300-ef3e-11e8-86a0-d54fe5fbe4bf.gif)

**Platform**
 - OS Version: Windows, MacOS
 - Browser Name: All
 - Browser Version: All

**Additional context**
We first noticed this when upgrading from 4.7 to 4.11
https://jira.infor.com/browse/LIME-4124
https://jira.infor.com/browse/LIME-4126

**Comment by clepore** [comment]
Duplicate of #1273

**Comment by tmcconechy** [comment]
@anhallbe I tested this out on the branch from https://github.com/infor-design/enterprise/pull/1297 and confirmed this is the same issue as this. Can no longer see the issue on your test after that. The fix is now going into 4.13.
